test/e2e/azure_test.go

```diff
@@ -41,7 +41,7 @@ var _ = Describe("Workload cluster creation", func() {
 		specName      = "create-workload-cluster"
 		namespace     *corev1.Namespace
 		cancelWatches context.CancelFunc
-		result       *clusterctl.ApplyClusterTemplateAndWaitResult
+		result        *clusterctl.ApplyClusterTemplateAndWaitResult
```

Comment:
nit: extra space
```suggestion
		result       *clusterctl.ApplyClusterTemplateAndWaitResult
```

Reply:
Noticed that this space gets added when I format the code in goland. I think it's okay to leave it. wdyt?